### PR TITLE
Created HTML barebones for landing page

### PR DESCRIPTION
**Pull request by cherishebojudah**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Code The Future</title>
+</head>
+<body>
+    <div>
+        <div>
+            <h1>Code The Future</h1>
+            <nav>
+                <ul>
+                   <li><a href="">Lorem</a></li>
```
Comment (Marienoir) [inline index.html]: Great work done @cherishebojudah 
For the link, I think you can put in the texts now since we already have a list of career paths.

Comment (cherishebojudah) [reply index.html]: Thank you. That would be done by my next commit.

**Review by Marienoir — APPROVED** [review]
Nice work

**Review by Onyinye101 — APPROVED** [review]
Looks good

**Pull request merged**